BREAKING.md

```diff
@@ -33,6 +34,9 @@ Please use `DriverErrorType.fileOverwrittenInStorage` instead of `OdspErrorType.
 ### absolutePath use in IFluidHandle is deprecated
 Rather than retrieving the absolute path, ostensibly to be stored, one should instead store the handle itself. To load, first retrieve the handle and then call `get` on it to get the actual object. Note that it is assumed that the container is responsible both for mapping an external URI to an internal object and for requesting resolved objects with any remaining tail of the external URI. For example, if a container has some map that maps `/a --> <some handle>`, then a request like `request(/a/b/c)` should flow like `request(/a/b/c) --> <some handle> --> <object> -->  request(/b/c)`.
 
+### ITelemetryBaseLogger now has a supportsTags property (not breaking)
+As part of moving telemetry tagging code to `common-definitions`, tagging support has been added to `ITelemetryBaseLogger` in the form of an optional boolean `supportsTags?: true` (additionally, tags are now just strings and `ITelemetryProperties` also accepts tagged properties). Currently, there is no further work necessary on any implementing code's part - old hosts will simply pass through this property undefined while our implemented logger in `telemetry-utils` handles tags in a different manner. But eventually this property will become required, so updating accordingly ASAP is encouraged.
```

Comment:
Thanks for adding this!  I'm going to reword it a bit to highlight action required of hosts integrating the next release.

```suggestion
### ITelemetryProperties may be tagged for privacy purposes
Telemetry properties on logs now may be tagged, meaning the property value may have the shape
`{ value: foo, tag: someString }` instead of merely a primitive value. Unwrapped/untagged values are still supported.
See the updated type definition of `ITelemetryProperties` in @fluidframework/common-definitions v 0.21.
`ITelemetryBaseLogger.send` should be updated to handle these tagged values.
See [this code](https://github.com/microsoft/FluidFramework/blob/main/packages/utils/telemetry-utils/src/logger.ts#L79-L107)
for an example of how to handle known tags, and update `supportsTags` to `true` when the logger properly handles tags.
Until then, in `ITelemetryBaseLogger.send` you may safely cast `event: ITelemetryProperties` to
`{ [index: string]: TelemetryEventPropertyType; }` and continue handling as-is,
since the FluidFramework will continue to handle tagged values itself unless `supportsTags` is `true`.
```

Reply:
Also update the link up top since I renamed the heading...

Reply:
PS - Let's try our best to complete the rest of this change in common-utils and telemetry-utils in this release. If not, it's actually ok, it would just mean that even if someone _does_ implement tag handling and sets `supportsTags: true`, we still will not hand out any tagged props 🤷‍♂️

Reply:
I think you will confuse people by this narrative. If there is no action required, then I'd remove it. If you want them to proactively do some work, then I think we need to put what exactly we assume hosts to do (and optionally - give timeframe, that it's not required now).

Either way, I think "core" description belongs to some readme.md, and breaking.md should be a reference to it with short description of the diff in expectations between FF versions.